Cargo.toml

```diff
@@ -6,5 +6,8 @@ members = [
   "iroha_client_cli",
   "iroha_macro",
   "iroha_macro/iroha_derive",
-  "iroha_network"
+  "iroha_network",
+  "modules/dex",
+  "apps/bridge_substrate_xclaim",
+  "modules/bridge_inner"
```

Comment:
you don't need to specify `modules` subfolder. what about app - is it mobile app?

Reply:
I think it helps to have a more clear repository structure. We have a risk to have a cargo mess in the future, so I desided to have a specific repository for module.